### DISCUSSION TITLE
web3modal QR-scanning works correctly on dev, don doesnt work on prod.

### DISCUSSION THREAD

**Discussion by EgorDovydenko** [WalletConnect/walletconnect-monorepo#3048] [Developer Support] [answered]
I use web3modal with wagmi. 
After QR code scanning on dev (localhost) all works correct: i go to Trust wallet, activate QR-scan, scanning process, get a popup for dApp connection, connect. But on production after QR scanning i dont see a dApp connection popup in my Trust wallet app. 

In my project on cloud.walletconnect Allowed Domains input are empty (_All origins are allowed by default_).

Here is my packages versions:
`
"@web3modal/ethereum": "2.6.2",
"@web3modal/react": "2.6.2",
"viem": "1.2.5",
"wagmi": "1.3.7"
 `.

**Comment by glitch-txs** [comment]
Next.js v12?

**Reply by EgorDovydenko** [reply]
13.0.3

**Comment by EgorDovydenko** [comment]
Any ideas?

**Reply by glitch-txs** [reply]
can you share your next.config.js? also if you have any public code you can share or error on console :)

**Reply by EgorDovydenko** [reply]
which part of the next.config.js are you interested in?

**Reply by glitch-txs** [reply]
Could you try setting swcMinify to false?

**Reply by EgorDovydenko** [reply]
`

const { withSentryConfig } = require('@sentry/nextjs');

/** @type {import('next').NextConfig} */
const fs = require('fs');
const nextConfig = {
  reactStrictMode: false,
  swcMinify: true,
  react: { useSuspense: false },
  publicRuntimeConfig: {
    NEXT_PUBLIC_API_HOST_INTERNAL: process.env.NEXT_PUBLIC_API_HOST_INTERNAL,
    NEXT_PUBLIC_API_HOST: process.env.NEXT_PUBLIC_API_HOST,
    USE_SENTRY: process.env.USE_SENTRY,
    SENTRY_DSN: process.env.SENTRY_DSN,
    NEXT_PUBLIC_SENTRY_DSN: process.env.NEXT_PUBLIC_SENTRY_DSN,
  },
  serverRuntimeConfig: {
    NEXT_PUBLIC_API_HOST_INTERNAL: process.env.NEXT_PUBLIC_API_HOST_INTERNAL,
    NEXT_PRIVATE_JWT: process.env.NEXT_PRIVATE_JWT,
    NEXT_PUBLIC_API_HOST: process.env.NEXT_PUBLIC_API_HOST,
    USE_SENTRY: process.env.USE_SENTRY,
    SENTRY_DSN: process.env.SENTRY_DSN,
    NEXT_PUBLIC_SENTRY_DSN: process.env.NEXT_PUBLIC_SENTRY_DSN,
    SENTRY_AUTH_TOKEN: process.env.SENTRY_AUTH_TOKEN,
  }
};

module.exports = withSentryConfig(
  nextConfig,
  { silent: true },
  { hideSourcemaps: true },
);

`

**Reply by EgorDovydenko** [reply, marked as answer]
`swcMinify: false` works! Thank you!
